pattern-library/sass/patterns/_buttons.scss

```diff
@@ -120,8 +114,8 @@ $btn-small-font-size: font-size(small);
     &.is-hovered,
     &:focus,
     &.is-focused {
-        border-color: palette(primary, accent);
-        background: palette(primary, accent);
+        border-color: $btn-focus-color;
+        background: $btn-focus-color;
         color: palette(primary, x-back);
     }
 
```

Comment:
I know this wasn't part of this PR, but do we have a ticket for improving the disabled button text color? It's way too low of a contrast, I'm sure it would trip the accessibility checker.